cmd/state/internal/cmdtree/cmdtree.go

```diff
@@ -113,6 +113,68 @@ func New(prime *primer.Values, args ...string) *CmdTree {
 
 	applyLegacyChildren(stateCmd, globals)
 
+	groups := []captain.CommandGroup{
```

Comment:
The group registration should live with the command itself. We don't want to start registering commands in multiple different ways.

Reply:
fwiw, I'd expect to see something like this:

```
Index: internal/captain/command.go
===================================================================
--- internal/captain/command.go	(revision ee63cd9bb4a9a38ccbd267bea9645c4004e3c8eb)
+++ internal/captain/command.go	(date 1603403020369)
@@ -42,7 +42,13 @@
 	out output.Outputer
 }
 
-func NewCommand(name, title, description string, out output.Outputer, flags []*Flag, args []*Argument, executor Executor) *Command {
+type CommandGroup string
+
+var (
+	PackagesGroup CommandGroup = "packages"
+)
+
+func NewCommand(name, title, description string, groups []CommandGroup, out output.Outputer, flags []*Flag, args []*Argument, executor Executor) *Command {
 	// Validate args
 	for idx, arg := range args {
 		if idx > 0 && arg.Required && !args[idx-1].Required {
```